features/step_definitions/kata.rb

```diff
@@ -0,0 +1,88 @@
+Given /^kata container has been installed successfully(?: in the #{QUOTED} project)?$/ do |ns|
+  kata_ns ||= "kata-operator"
+  step %Q/I switch to cluster admin pseudo user/
+  unless namespace(kata_ns).exists?
+    @result = user.cli_exec(:create_namespace, name: kata_ns)
+    raise "Failed to create namespace #{kata_ns}" unless @result[:success]
+  end
+  project(kata_ns)
+  iaas_type = env.iaas[:type] rescue nil
+  raise "Kata installation only supports GCE platform currently." if iaas_type != 'gcp'
+  # check to see if kata already exists
+  unless kata_config('example-kataconfig').exists?
+    # setup service account
+    role_yaml = "https://raw.githubusercontent.com/openshift/kata-operator/release-4.6/deploy/role.yaml"
```

Comment:
Hard code to 4.6 ? Will it supported in 4.7 ?

Reply:
@liangxia how 4.7 works is still not cleared, that's the only working model.

Reply:
@liangxia I've updated the file path to be keyed off by the master version, so that should make it more flexible.